htdocs/core/lib/functions.lib.php

```diff
@@ -15764,6 +15766,8 @@ function show_actions_messaging($conf, $langs, $db, $filterobj, $objcon = null,
 			$sql .= " LEFT JOIN " . MAIN_DB_PREFIX . "commande as o ON a.fk_element = o.rowid AND a.elementtype = 'order'";
 		} elseif (is_object($filterobj) && get_class($filterobj) == 'Expedition') {
 			$sql .= " LEFT JOIN " . MAIN_DB_PREFIX . "expedition as s ON a.fk_element = s.rowid AND a.elementtype = 'shipping'"; // CHANGED: Table 'expedition' et alias 's', elementtype 'shipping'
+		} elseif (is_object($filterobj) && get_class($filterobj) == 'Propal') {
+			$sql .= " LEFT JOIN " . MAIN_DB_PREFIX . "propal as o ON a.fk_element = o.rowid AND a.elementtype = 'propal'";
```

Comment:
You should use an exact join because we want the actioncomm that are linked to the propal. If not we don't want them, so a left join is not necessary here, a
" INNER JOIN " 
should give the same result with better performances.